lib/utils.js

```diff
@@ -1,6 +1,18 @@
 import NativePromise from 'native-promise-only'
+import pkgJson from '../package.json'
+
 const Promise = window.Promise || NativePromise
 
+export function addUserAgent (headers, appId) {
+  const originalUserAgent = headers && headers['User-Agent'] ? headers['User-Agent'] : ''
+  const zafSdkUserAgentString = `zendesk_app_framework_sdk/sdk_version:${pkgJson.version}/app_id:${appId}`
+  const userAgent = originalUserAgent ? `${originalUserAgent} ${zafSdkUserAgentString}` : zafSdkUserAgentString
```

Comment:
nit: consider ```const userAgent = `${originalUserAgent} ${zafSdkUserAgentString}`.trim();```

Reply:
As above, optional chaining is not possible with node 12 =(